lib/task/migrate/arUpgradeSqlTask.class.php

```diff
@@ -246,6 +250,196 @@ protected function execute($arguments = [], $options = [])
+                $page->slug = $slug;
+                $page->sourceCulture = 'en';
+                $page->save();
+            }
+        }
+    }
+
+    /**
+     * For all standard static pages, reset translations in the database
+     * to match the current AtoM default values.
+     */
+    protected function resetChangedStaticPages()
+    {
+        $changes = $this->getStaticPageTranslationResetValues();
+
+        foreach ($changes as $slug => $translationData) {
+            $page = QubitStaticPage::getBySlug($slug);
+
+            if (null !== $page) {
+                foreach ($translationData as $culture => $translations) {
+                    if (isset($translations['title'])) {
+                        $page->setTitle($translations['title'], ['culture' => $culture]);
+                    }
+
+                    if (isset($translations['content'])) {
+                        $page->setContent($translations['content'], ['culture' => $culture]);
+                    }
+                }
+            }
+
+            $page->save();
+        }
+    }
+
+    /**
+     * Generate an array containing the static page translation values needed
+     * to change translations in the database to the current AtoM default values.
+     *
+     * Example of the structure:
+     *
+     *  [
+     *    'someslug' => [
+     *      'en' => [
+     *        'title' => 'Default English Title'
+     *      ]
+     *    ],
+     *    'anotherslug' => [
+     *      'fr' => [
+     *        'content' => 'Contenu français par défaut'
+     *      ]
+     *    ]
+     *  ]
+     *
+     * @return array
+     */
+    protected function getStaticPageTranslationResetValues()
+    {
+        $changes = [];
+
+        foreach ($this->parseStaticPageFixtureData() as $slug => $translationData) {
+            $page = QubitStaticPage::getBySlug($slug);
+
+            // Only check if page still exists in the database (the privacy page can be deleted)
+            if (null !== $page) {
+                $pageChanges = [];
+
+                // Check each standard translation against the database version of it
+                foreach ($translationData as $culture => $translations) {
+                    $pageChanges[$culture] = [];
+
+                    // Check page title
+                    if ($translations['title'] != $page->getTitle(['culture' => $culture])) {
+                        $pageChanges[$culture]['title'] = $translations['title'];
+                    }
+
+                    // Check page content
+                    if ($translations['content'] != $page->getContent(['culture' => $culture])) {
+                        $pageChanges[$culture]['content'] = $translations['content'];
+                    }
+
+                    // This translation was unchanged so we can remove it from the array of page changes
+                    if (empty($pageChanges[$culture])) {
+                        unset($pageChanges[$culture]);
```

Comment:
I think if you remove: `$pageChanges[$culture] = [];` from the beginning of this loop it makes this "unset()" unnecessary?